samples/csharp/MicrosoftGraphSDK/Assignment.cs

```diff
@@ -36,6 +36,31 @@ public class Assignment
             }
         }
 
+        /// <summary>
+        /// Get all the assignments from the class
+        /// </summary>
+        /// <param name="client"></param>
+        /// <param name="classId"></param>
+        /// <returns>EducationAssignment</returns>
```

Comment:
return type must match what you have in line 45

Reply:
Thanks for your comment, changes have been added to the latest commit

Reply:
Thanks for your comment, changes have been added to the latest commit